versioned_docs/version-2.x/appendices/migration-guides/index.mdx

```diff
@@ -256,6 +256,23 @@ your website.
 
 :::
 
+### Performance: remove chunks preloading
+
+We've
+[removed the chunks preloading feature](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2120)
+as it led to blocking network requests that could reduce the
+
+<abbr title="Largest Contentful Paint">LCP</abbr> /<abbr title="First Contentful Paint">
+  FCP
+</abbr> of your app. From our analysis it improves the performance of the
+application and is a good default.
+
+We invite you to test the impact of such a change (i.e: run a Core Web Vitals
+test on your application, and compare the results) to confirm this.
```

Comment:
```suggestion
We invite you to test the impact of this change (i.e: run a Core Web Vitals
test on your application and compare the results to confirm this).
```

It could also be good to invite integrators to contact us if they have any feedback?

Reply:
Applied (commit amended). Thanks.